app/controllers/books_controller.rb

```diff
@@ -2,7 +2,11 @@ class BooksController < ApplicationController
   before_action :find_book, except: :index
 
   def index
-    @pagy, @books = pagy(Book.newest, items: Settings.pagy.digit_10)
+    if params[:term]
+      @pagy, @books = pagy(search_books.newest, items: Settings.pagy.digit_10)
+    else
+      @pagy, @books = pagy(Book.newest, items: Settings.pagy.digit_10)
+    end
```

Comment:
```suggestion
    books = Books.search_by_tilte.newest
    @pagy, @books = pagy books, items: Settings.pagy.digit_10
```

xử lý có param search hay ko ở scope luôn em nhé